admin/data/sql/migrations/20190621143742.undo.create-pupil-results-view.sql

```diff
@@ -0,0 +1 @@
+DROP VIEW [mtc_admin].[vewPupilsResults]
```

Comment:
Should be DROP VIEW IF EXISTS to be idempotent.

Reply:
@jon-shipley changed